FlipperZero/Core/View/DesignPlaygroundView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  DesignPlaygroundView.swift
+//  Core
+//
+//  Created by Yachin Ilya on 25.03.2021.
+//
+
+import SwiftUI
+// swiftlint:disable closure_body_length indentation_width let_var_whitespace
```

Comment:
Let's disable `closure_body_length` for the `View` directory. Other violations should be fixed instead of disabled

Reply:
Have to remove explicit path to config in Actions to enable per directory configs be read automatically. Currently there is a `closure_body_length` warning here in `lint` job but it isn't risen in Xcode.

Reply:
Sorry I want to take it back. Let's temporarily keep `// swiftlint:disable closure_body_length` here, rollback config changes and update config in #31 or another separate PR

Reply:
Moved separate config for `View` to #31